curriculum/challenges/english/25-front-end-development/lab-lunch-picker-program/66db529d37ad966480ebb633.md

```diff
@@ -0,0 +1,153 @@
+---
+id: 66db529d37ad966480ebb633
+title: Build a Lunch Picker Program
+challengeType: 14
+dashedName: build-a-lunch-picker-program
+---
+
+# --description--
+
+In this lab, you'll build a program to assist in choosing lunch options. You'll work with two arrays: one for children's food choices and one for adults. You'll add and remove items from these arrays and then randomly select a meal from each. Finally, you'll log the results to the console.
+
+**Objective:** Fulfill the user stories below and get all the tests to pass to complete the lab. 
+
+**User Stories:**
+
+1. You should create a `kidsMeal` variable and assign it an array with at least three strings in it.
+2. You should create an `adultsMeal` variable and assign it an array with at least three strings in it.
+3. Once items have been added to the `kidsMeal`, you should use the appropriate array method to remove the last item.
+4. Once items have been added to the `adultsMeal`, you should use the appropriate array method to remove the first item. 
+5.  You should use the appropriate `Math` methods to generate a random number between `0` to the length of the `kidsMeal` array. Assign that result to a variable called `randomIndexKids`.
+6. You should use the appropriate `Math` methods to generate a random number between `0` to the length of the `kidsMeal` array. Assign that result to a variable called `randomIndexAdults`.
+7.  Log the following messages to the console:
+
+- `"Today kids should eat:"` followed by the selected meal to the console.
+- `"Today adults should eat:"` followed by the selected meal to the console.
```

Comment:
I worry a little that people will think they need to log the quotes to the console - but I'm fine with it like this for now. If people are getting confused, maybe we can change it later.